[rospy.client][INFO] 2022-02-14 16:03:35,450: init_node, name[/rostopic_4026_1644854615389], pid[4026]
[xmlrpc][INFO] 2022-02-14 16:03:35,450: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2022-02-14 16:03:35,450: Started XML-RPC server [http://FM00351-u:37577/]
[rospy.init][INFO] 2022-02-14 16:03:35,450: ROS Slave URI: [http://FM00351-u:37577/]
[rospy.impl.masterslave][INFO] 2022-02-14 16:03:35,450: _ready: http://FM00351-u:37577/
[rospy.registration][INFO] 2022-02-14 16:03:35,451: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2022-02-14 16:03:35,451: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2022-02-14 16:03:35,551: registered with master
[rospy.rosout][INFO] 2022-02-14 16:03:35,551: initializing /rosout core topic
[rospy.rosout][INFO] 2022-02-14 16:03:35,555: connected to core topic /rosout
[rospy.simtime][INFO] 2022-02-14 16:03:35,556: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2022-02-14 16:03:35,619: topic[/Robotiq2FGripperRobotOutput] adding connection to [http://FM00351-u:44709/], count 0
[rospy.internal][INFO] 2022-02-14 16:03:35,833: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2022-02-14 16:04:31,171: signal_shutdown [signal-2]
[rospy.internal][INFO] 2022-02-14 16:04:31,175: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2022-02-14 16:04:31,176: topic[/Robotiq2FGripperRobotOutput] removing connection to http://FM00351-u:44709/
[rospy.internal][WARNING] 2022-02-14 16:04:31,178: Unknown error initiating TCP/IP socket to FM00351-u:42091 (http://FM00351-u:44709/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2022-02-14 16:04:31,178: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 846, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2022-02-14 16:04:31,179: signal-2
[rospy.core][INFO] 2022-02-14 16:04:31,287: signal_shutdown [atexit]
